vtex-apps/store-components
# Feat/ send vtex:share event on social buttons

#### What problem is this solving?

Send the new `share` event for the GTM app

#### How to test it?

* Go to this [branch](https://github.com/vtex-apps/store-components/tree/feat/ga4-send-share-event)
* Link store-components to workspace
* Switch to [branch with new GTM version events](https://github.com/vtex-apps/google-tag-manager/tree/feat/settings-merge-ua-events) and link on workspace
* Click and open a product, after choose one of share options 
* Check on `window.dataLayer` if the item has the correct attributes


[Workspace](https://natandevuamerge--storecomponents.myvtex.com/wood-clock/p)

#### Screenshots or example usage:

<img width="1437" alt="Captura de Tela 2023-04-06 às 18 00 31" src="https://user-images.githubusercontent.com/36740164/230493118-1292385b-9897-4108-8163-4051b27ecb35.png">


#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/gcajW7oKirCdW/giphy.gif)
